ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,257 @@
+    "ListElementCode": "situationDate",
+    "FieldType": "Date",
+    "ElementLabel": "What date did they start staying here?",
+    "FieldShowCondition": "livingSituation=\"*\"",
+    "HintText": "For example, 31 3 2018",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "keyDates",
+    "FieldType": "TextArea",
+    "ElementLabel": "Key dates for this child",
+    "HintText": "List any events HMCTS will need to take into account when scheduling hearings. For example, child starting primary school or taking GCSEs.",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "careAndContact",
+    "FieldType": "TextArea",
+    "ElementLabel": "Brief summary of care and contact plan",
+    "HintText": "For example, place baby in local authority foster care until further assessments are completed. Supervised contact for parents will be arranged.",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "adoption",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Are you considering adoption at this stage?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "placementOrderApplication",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Are you submitting an application for a placement order?",
+    "FieldShowCondition": "adoption=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "placementCourt",
+    "FieldType": "TextArea",
+    "ElementLabel": "Which court are you applying to?",
+    "FieldShowCondition": "placementOrderApplication=\"Yes\"",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "mothersName",
+    "FieldType": "Text",
+    "ElementLabel": "Mother's full name",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "fathersName",
+    "FieldType": "Text",
+    "ElementLabel": "Father's full name",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "fathersResponsibility",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "FathersResponsibilityList",
+    "ElementLabel": "Does the father have parental responsibility?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "socialWorkerName",
+    "FieldType": "Text",
+    "ElementLabel": "Name of social worker",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "partyChild",
+    "ListElementCode": "socialWorkerTel",
```

Comment:
should this follow the TelephoneNumber reform type?

Reply:
I considered this but decided it was an unnecessary change, what do you think?

Reply:
I think all telephone numbers should be the reform type. A discussion we might need to have @geek-soft @Jamesnelson117

Reply:
@geek-soft has suggested we should use reform telephoneNumber type. Making the change in next commit...